gradle-plugin/src/main/groovy/mozilla/telemetry/glean-gradle-plugin/GleanGradlePlugin.groovy

```diff
@@ -34,7 +34,7 @@ class GleanMetricsYamlTransform extends ArtifactTransform {
 @SuppressWarnings("GrPackage")
 class GleanPlugin implements Plugin<Project> {
     // The version of glean_parser to install from PyPI.
-    private String GLEAN_PARSER_VERSION = "1.23.0"
+    private String GLEAN_PARSER_VERSION = "1.24.0"
```

Comment:
Did you try to build locally and see if docs were updated? Because they are not included in this PR.

Reply:
Yes, I built locally and nothing changed. That is why no changes here.